pkg/apis/options/options.go

```diff
@@ -85,22 +85,23 @@ type Options struct {
 
 	// These options allow for other providers besides Google, with
 	// potential overrides.
-	ProviderType                       string `flag:"provider" cfg:"provider"`
-	ProviderName                       string `flag:"provider-display-name" cfg:"provider_display_name"`
-	OIDCIssuerURL                      string `flag:"oidc-issuer-url" cfg:"oidc_issuer_url"`
-	InsecureOIDCAllowUnverifiedEmail   bool   `flag:"insecure-oidc-allow-unverified-email" cfg:"insecure_oidc_allow_unverified_email"`
-	InsecureOIDCSkipIssuerVerification bool   `flag:"insecure-oidc-skip-issuer-verification" cfg:"insecure_oidc_skip_issuer_verification"`
-	SkipOIDCDiscovery                  bool   `flag:"skip-oidc-discovery" cfg:"skip_oidc_discovery"`
-	OIDCJwksURL                        string `flag:"oidc-jwks-url" cfg:"oidc_jwks_url"`
-	LoginURL                           string `flag:"login-url" cfg:"login_url"`
-	RedeemURL                          string `flag:"redeem-url" cfg:"redeem_url"`
-	ProfileURL                         string `flag:"profile-url" cfg:"profile_url"`
-	ProtectedResource                  string `flag:"resource" cfg:"resource"`
-	ValidateURL                        string `flag:"validate-url" cfg:"validate_url"`
-	Scope                              string `flag:"scope" cfg:"scope"`
-	Prompt                             string `flag:"prompt" cfg:"prompt"`
-	ApprovalPrompt                     string `flag:"approval-prompt" cfg:"approval_prompt"` // Deprecated by OIDC 1.0
-	UserIDClaim                        string `flag:"user-id-claim" cfg:"user_id_claim"`
+	ProviderType                       string   `flag:"provider" cfg:"provider"`
+	ProviderName                       string   `flag:"provider-display-name" cfg:"provider_display_name"`
+	ProviderCAFiles                    []string `flag:"provider-ca-file" cfg:"provider_ca_file"`
```

Comment:
When this is in the configuration file, it should be pluralised, so this should be 
```suggestion
	ProviderCAFiles                    []string `flag:"provider-ca-file" cfg:"provider_ca_files"`
```